Code/Legacy/CryCommon/IShader.h

```diff
@@ -770,7 +770,7 @@ _MS_ALIGN(16) struct SSkinningData
     SSkinningData*          pNextSkinningData;          // List to the next element which needs SW-Skinning
 } _ALIGN(16);
 
-struct SRenderObjData
+struct _MS_ALIGN(16) SRenderObjData
```

Comment:
`alignas(16)` can be used here

Reply:
I will do another pass and remove that macro... there are two different macros that need to be used to be compatible with MS and Clang (this was clearly pre-C++11)